gravatar/src/main/java/com/gravatar/services/interceptors/SdkVersionInterceptor.kt

```diff
@@ -0,0 +1,16 @@
+package com.gravatar.services.interceptors
+
+import com.gravatar.BuildConfig
+import okhttp3.Interceptor
+import okhttp3.Response
+
+internal class SdkVersionInterceptor : Interceptor {
+    override fun intercept(chain: Interceptor.Chain): Response {
+        return chain.proceed(
+            chain.request().newBuilder()
+                .addHeader("X-Platform", "Android")
```

Comment:
❓ 💡 Can it make sense to diffenciate between release and debug versions?

Reply:
hmm good question. I will ask iOS folks about that. 

I assume you want to be able to filter "our" usage of the SDK, right? Another solution could be modifying how we read the last tag. Currently, it's `git describe --tags --abbrev=0` but if we got rid of the `--abbrev=0`. That way only the build from the 1.1.0 tag will have a clean version without the commit suffix.

Reply:
> Currently, it's git describe --tags --abbrev=0 but if we got rid of the --abbrev=0. That way only the build from the 1.1.0 tag will have a clean version without the commit suffix.

TIL - Sounds good!